lib/carrierwave/storage.rb

```diff
@@ -2,8 +2,9 @@
 require "carrierwave/storage/file"
 
 begin
-  require "fog" unless defined?(::Fog)
+  require "fog/core"
 rescue LoadError
+  require "fog" unless defined?(::Fog)
```

Comment:
Yeah, there used to be a `rescue LoadError` after `require "fog"` here. I’m not sure why, but I can add it back.